src/Common/Common.csproj

```diff
@@ -1,4 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
This was moved from a property group with a label set below but the label did not also move